common/pom.xml

```diff
@@ -37,6 +38,18 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
```

Comment:
yeah, "provided" works fine for our microservices, since they're all including some form of servlet container.